treeherder/model/migrations/0021_add_jobdetail_model.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import treeherder.model.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0020_update_job_name_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='JobDetail',
+            fields=[
+                ('id', treeherder.model.fields.BigAutoField(serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=512, null=True)),
```

Comment:
Yeah I'm aware of that. I deleted the record of the migration from the database manually from stage while testing (along with the table). I'll take care of Heroku later (by more or less doing the same thing).

I really shouldn't have landed this change to master, period, since it quickly became evident that this change needed a lot more testing in a real environment -- I had tested locally with a subset of data from stage, but that evidently wasn't enough.

But yes, in another circumstance that would be good advice. :)